lib/dbMapping.js

```diff
@@ -87,3 +87,73 @@ export const APPLICATION_STATE = [
   'NFI Check',
 ];
 export const GRANT_AMOUNT = ['0', '5000', '10000'];
+
+export const BUSINESS_CATEGORIES = {
+  Retail: [
+    'Clothing stores and tailors',
+    'Homeware stores',
+    'Tobacco and vape shops',
+    'Electronic goods and mobile phone shops',
+    'Markets (Non-Essential)',
+    'Car and other vehicle showrooms',
+    'Car washes',
+    'Auction houses',
+    'Other Non-Essential Retail',
+  ],
+  Hospitality: ['Restaurants', 'Pubs', 'Bars', 'Cafes', 'Other Hospitality'],
```

Comment:
Really picky but can we make the formatting consistent? i.e. each array item on a new line (took me a little bit to parse this line and realise it was same as the others.)

Reply:
I know that could be annoying, but we are using https://prettier.io/ for formatting and it's all automatic. They are not in the same format for a matter of print width https://prettier.io/docs/en/rationale.html#print-width :(